client/asset/btc/electrum.go

```diff
@@ -170,18 +176,18 @@ func (btc *ExchangeWalletElectrum) Sweep(address string, feeSuggestion uint64) (
 	return newOutput(&txHash, 0, 0 /* ! */), nil
 }
 
-// override feeRate to avoid unnecessary conversions and btcjson types.
-func (btc *ExchangeWalletElectrum) feeRate(_ RawRequester, confTarget uint64) (uint64, error) {
-	satPerKB, err := btc.ew.wallet.FeeRate(btc.ew.ctx, int64(confTarget))
+func (btc *ExchangeWalletElectrum) walletFeeRate(ctx context.Context, confTarget uint64) (uint64, error) {
```

Comment:
No changes were really needed in electrum.go, but I wanted to remove the unused RawRequester junk from the `ExchangeWalletElectrum` method, doing the wrapping inside the `ElectrumWallet` constructor.